README.md

```diff
@@ -34,22 +37,81 @@ Once installed, you will have two command-line scripts. Ask for more info using
+
+
+### isomorphous data, please
+
+METEOR is only for isomorphous difference maps, meaning the lattices/symmetries of the _native_ and _derivative_ datasets are comparable. If you need to compare non-isomorphous lattices, check out [matchmaps](https://github.com/rs-station/matchmaps).
+
+
+## command-line details
+
+`meteor` provides two command-line scripts that most users will want. If you prefer working in Jupyter notebooks or want to develop against `meteor`'s library, refer to the [API documentation](https://rs-station.github.io/meteor/).
+
+Both of `meteor`'s scripts generate difference maps (MTZs). We recommend starting with `meteor.diffmap`. This script applies k-weighting then TV-denoising, and picks parameters for both by maximizing negentropy. It's relatively fast to run (expect about a minute) and has fewer nobs to turn. After you've tried `meteor.diffmap`, you can give `meteor.phaseboost` a try. This script iteratively applys TV denoising, and adjusts the phases of the _derivative_ data to try and produce a denoised map. At the end, it applies a k-weighting and TV denoising pass. It often results in slightly better maps, at the cost of additional compute (many minutes). We also have noticed the algorithm can become unstable in rare circumstances. If you think that might be happening to you, please get in touch.
+
+Note that individual steps in both of these scripts can be turned off or modified using command-line flags, as described below.
+
+One note: in the lingo we adope, we compute _derivative_ minus _native_ maps. Initial phases typically from a model of the _native_ data, computed from a CIF/PDB model. Usually the derivative data are ligand bound, time-resolved-activated, or similar... but in the end, the use case is defined by you!
+
+### meteor.diffmap
+
+Compute difference maps, including k-weighting and TV-denoising options. To see the full help, just run `meteor.diffmap -h`. Here are some examples that highlight common use cases:
+
+Computing a k-weighted, TV-denoised diffmap, with `meteor` making some smart default choices for me:
+```
+meteor.diffmap derivative.mtz native.mtz -s native.pdb
+```
+note that the order of `derivative.mtz` and `native.mtz` matters!
+
+Suppose I have some non-standard column names. Using `gemmi mtz <my.mtz>` might be smart to find out what they should be; then,
+```
+meteor.diffmap derivative.mtz --derivative-amplitude-column F_ON --derivative-uncertainty-column SIGF_ON native.mtz -s native.pdb
+```
+or, equivalently,
+```
+meteor.diffmap derivative.mtz --da F_ON --du SIGF_ON native.mtz -s native.pdb
+```
+
+what if I want to compute a k-weighted map, with k-parameter of `0.05`, and no TV?
+```
+meteor.diffmap derivative.mtz native.mtz -s native.pdb --kweight-mode fixed --kweight-parameter 0.05 --tv-denoise-mode none 
+```
+
+To note and remember:
+
```

Comment:
Somewhere we should explain that it one wants to do a Fo-Fo map, they should provide the same phase twice. A good place could be the diffmap script or the compute_difference_map function

Reply:
hmm what do you mean? right now the CLI only takes phases from a PDB model, never from an MTZ.